modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -16,41 +16,27 @@ Some administrators prefer to use static IP addresses so that each node's IP add
 ====
 endif::[]
 
-Preparing the bare metal node requires executing the following procedure from the provisioner node.
-
 .Procedure
 
-. Get the `oc` binary, if needed. It should already exist on the provisioner node.
-+
-[source,bash]
-----
-[kni@provisioner ~]$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/openshift-client-linux-$VERSION.tar.gz | tar zxvf - oc
-----
-+
-[source,bash]
-----
-[kni@provisioner ~]$ sudo cp oc /usr/local/bin
-----
-
 . Power off the bare metal node via the baseboard management controller and ensure it is off.
 
 . Retrieve the user name and password of the bare metal node's baseboard management controller. Then, create `base64` strings from the user name and password. In the following example, the user name is `root` and the password is `calvin`.
```

Comment:
Per the Supplementary Style Guide, [user-replaced values](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values) should be surrounded by angle brackets and italicized. 

Suggested rewrite:
`_<root>_` and the password is `_<calvin>_`